bin/gwdetchar-scattering

```diff
@@ -347,6 +347,9 @@ for i, channel in enumerate(sorted(alldata[0].keys())):
     else:
         effdt = efficiencypc/deadtimepc
     logger.info("Efficiency/Deadtime: %.2f" % effdt)
+    
+    if abs(scatter_segments[channel].active):    
+          actives.append(scatter_segments[channel].active)
```

Comment:
@siddharth101, after looking into it a bit, I think it's better to use the `extend` method:

```suggestion
          actives.extend(scatter_segments[channel].active)
```

The difference is a bit subtle. If you have a `list` object, `list.append()` adds a single element to the end of the list, while `list.extend()` takes in another list and adds _all_ of its elements to the end of the first. The upshot to this is that using `actives.append` will leave you with a list of `SegmentList`s, while `actives.extend` gives a single `SegmentList`. This will make it possible to clean up the code block below.

Reply:
Thats nice.  Will make this change. This will help get rid of one more line of code.